drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -138,12 +138,19 @@ removeCollectionQuestion=Do you want to continue and remove the collection?
 removeCollectionWarningText=This action cannot be undone.
 #
 settings=Settings
+name=Name
+type=Type
+statelessLabel=Stateless Session
+dmnModelLabel=DMN model
+dmnNamespaceLabel=DMN namespace
```

Comment:
Not sure if it makes sense to translate these words to be honest

Reply:
@danielezonca Yes, we can consider them as technical terms, so fine for me to don't translate them. Done.